app/controllers/conversations_controller.rb

```diff
@@ -0,0 +1,11 @@
+class ConversationsController < ApplicationController
+  def show
+    @conversation = Conversation.find(params[:id])
+
+    if @conversation.sender_id == current_user.id
+      @friend = User.find(@conversation.receiver_id)
+    else
+      @friend = User.find(@conversation.sender_id)
```

Comment:
you can improve this as @friend = @conversation.sender_id == current_user.id ? current_user.receiver : current_user